pages/about/jobs/staff-academic-jobs/index.vue

```diff
@@ -0,0 +1,170 @@
+<script setup>
+// HELPERS
+import _get from 'lodash/get'
+import fixUri from '../utils/fixUri'
+import removeTags from '../utils/removeTags'
+
+// GQL
+import JOB_OPPORTUNITIES_LIST from '../gql/queries/JobOpportunitiesList.gql'
+
+const { $graphql } = useNuxtApp()
+
+const { data } = await useAsyncData('job-opportunities-list', async () => {
```

Comment:
add error handling like the slug pages, check previewmode PR for additional changes